CONTRIBUTING.md

```diff
@@ -1,7 +1,28 @@
-# Contributing
+# Contributing to panther-analysis
 
-Please follow the [Code of Conduct](https://github.com/panther-labs/panther-analysis/blob/main/CODE_OF_CONDUCT.md)
-in all of your interactions with the project.
+Thank you for your interest in contributing to Panther's open-source ruleset!  We value and appreciate all contributions, whether they are bug reports, feature requests, or detection rule contributions.
+
+## What Makes a Good Detection?
+
+Please familiarize yourself with these helpful resources on writing high-quality Panther rules.  Especially excellent contributions will be considered for seasonal prizes!
```

Comment:
sugg: end first sentence (after rules) with a colon (:)

and put the second sentence after the list of resources